tests/sdk/driver/test_ce_runtime.py

```diff
@@ -990,58 +990,101 @@ def test_happy_path(
             page_size=None,
             page_token=None,
             workspace=None,
+            max_age=None,
+            state=None,
         )
         assert runs == wf_runs
 
     @pytest.mark.parametrize(
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏These looks like two very similar tests. Maybe an area for improvement another time.